packages/cli/src/cli.ts

```
@@ -26,6 +26,11 @@ import {
   fetchPartners,
   searchTanStackDocs,
 } from './discovery.js'
+import {
+  searchYouCom,
+  getYouComMCPConfig,
```

ESLint

[error] Member 'getYouComMCPConfig' of the import declaration should be sorted alphabetically.

(sort-imports)